sessionctx/variable/varsutil.go

```diff
@@ -655,6 +655,15 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 			}
 		}
 		return formatVal, nil
+	case TiDBMetricSchemaStep, TiDBMetricSchemaRangeDuration:
+		v, err := strconv.ParseInt(value, 10, 64)
+		if err != nil {
+			return value, ErrWrongValueForVar.GenWithStackByArgs(name)
+		}
+		if v < 10 || v > 60*60*60 {
```

Comment:
You can use `checkUInt64SystemVar(name, value, 10, 60*60*60, vars)`

Reply:
`checkUInt64SystemVar` will not return error when the `v` is not valid.